modules/mcuboot/CMakeLists.txt

```diff
@@ -109,6 +114,16 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       )
   endif()
 
+  if (CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY)
+    # By passing this value we don't require the user to pass any overlay files
+    # to the MCUboot child image for storing the secondary partition in
+    # external flash.
+    add_overlay_config(
+      mcuboot
+      ${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/ext_flash_mcuboot_secondary.conf
+      )
```

Comment:
should this really be passed as a Kconfig fragment ?
Why not simply define `-DCONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=y` when building the child image ?

Reply:
This solution is more visible for the user. This way the config passed to the child image can be seen from the cmake output.

Reply:
discussed offline.
Accepted to keep for 1.7.